trompace/queries/mediaobject.py

```diff
@@ -0,0 +1,64 @@
+# Generate GraphQL queries for queries pertaining to media objects.
+from trompace.exceptions import UnsupportedLanguageException, NotAMimeTypeException
+from trompace.queries.templates import format_query
+from trompace import StringConstant, _Neo4jDate, filter_none_args
+from trompace.constants import SUPPORTED_LANGUAGES
+
+
+def query_mediaobject(identifier: str=None, name: str=None, description: str=None, date: str=None, creator: str=None, contributor: str=None, format_: str=None,\
+ encodingFormat: str=None, source: str=None, subject: str=None, contentUrl:str=None, language: str=None, title:str=None):
+
+    """Returns a mutation for creating a media object object
```

Comment:
did you just copy this? Please make sure that documentation makes sense for the method that it's connected to, otherwise we may as well not write it.